### PR TITLE
Update package_version gatherer doc

### PR DESCRIPTION
**Pull request by nelsonkopliku**
Updating doc based on changes made here https://github.com/trento-project/agent/pull/181

### CODE REVIEW

File: guides/gatherers.md [hunk 1]
```diff
@@ -327,14 +344,20 @@ values:
...
 Example arguments:
 
-| Name                 | Return value                                                 |
-| :------------------- | :----------------------------------------------------------- |
-| `package_name`       | a string containing the installed version of the rpm package |
-| `package_name,2.4.5` | an integer with a value of `-1`, `0` or `-1` (see above)     |
+| Name                 | Return value                                                                  |
+| :------------------- | :---------------------------------------------------------------------------- |
+| `package_name`       | a list containing information about the installed versions of the rpm package |
+| `package_name,2.4.5` | an integer with a value of `-1`, `0` or `-1` (see above)                      |
```
Comment (angelabriel) [inline guides/gatherers.md:360]: typo - the values are '-1', '0' and '**1**'

Comment (nelsonkopliku) [reply guides/gatherers.md]: Thanks! Totally missed :sweat_smile:

File: guides/gatherers.md [hunk 2]
```diff
@@ -327,14 +344,20 @@ values:
 expectations:
   - name: compare_package_corosync
     expect: facts.compare_package_corosync == values.greater_than_installed
+  
+  - name: compare_package_corosync
+    expect: facts.package_corosync.first().version == "2.4.5"
+
+  - name: sbd_version_same_on_all_hosts
+    expect_same: facts.package_sbd.firts().version
```
Comment (angelabriel) [inline guides/gatherers.md:352]: Sorry, me again :smiley: 
typo - firts instead of first

Comment (nelsonkopliku) [reply guides/gatherers.md]: super :rocket:

File: guides/gatherers.md [hunk 3]
```diff
@@ -295,14 +295,23 @@ Example output (in Rhai):
 **Argument required**: yes.
 
 This gatherer supports two usecases:
- - get the version as a string of the specified package (useful to check if the installed version of a package is the same across multiple nodes)
- - compare a given version string against the installed version string of a given package.
+ - get information about the installed versions of the specified package
```
Comment (arbulu89) [inline guides/gatherers.md:298]: ocd: Let's add a dot at the end of the sentence.

File: guides/gatherers.md [hunk 4]
```diff
@@ -327,14 +344,20 @@ values:
 expectations:
   - name: compare_package_corosync
     expect: facts.compare_package_corosync == values.greater_than_installed
+  
+  - name: compare_package_corosync
+    expect: facts.package_corosync.first().version == "2.4.5"
```
Comment (arbulu89) [inline guides/gatherers.md:349]: We could put this `2.4.5` in the `values` I guess

**Review by arbulu89 — APPROVED** [review]
Great!

**Review by fabriziosestito — APPROVED** [review]

**Review by EMaksy — COMMENTED** [review]
LGTM :D

**Pull request merged**